sites/site-utilities/src/definitions/fast-components/fast-accordion.definition.ts

```diff
@@ -0,0 +1,35 @@
+import { WebComponentDefinition } from "@microsoft/fast-tooling/dist/data-utilities/web-component";
+import { DataType } from "@microsoft/fast-tooling";
+
+export const fastAccordionDefinition: WebComponentDefinition = {
+    version: 1,
+    tags: [
+        {
+            name: "fast-accordion",
+            description: "The FAST accordion element",
+            attributes: [
+                {
+                    name: "expand-mode",
+                    description: "The expand mode attribute",
+                    type: DataType.string,
+                    values: [
+                        {
+                            name: "single",
+                        },
+                        {
+                            name: "multi",
+                        },
+                    ],
+                    default: "multi",
+                    required: false,
+                },
+            ],
+            slots: [
```

Comment:
Does accordion not have a default slot?

Reply:
Nope only a slot for item

Reply:
Do the accordion items automatically set their slot to item by any chance?

Reply:
yes they do